src/components/App.js

```diff
@@ -0,0 +1,12 @@
+import '../app.css';
+import Display from './Display';
+import ButtonPannel from './ButtonPanel';
+
+const App = () => (
+  <div>
+    <Display />
+    <ButtonPannel />
+  </div>
```

Comment:
- [ ] [REQUIRED] It would be better to have the children components wrapped in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/)) just like you did in the ButtonPanel component. This is because this helps you group a list of children without adding extra nodes to the DOM.

Reply:
I added now
Thank you for reviewing